main.go

```diff
@@ -197,6 +198,8 @@ func main() {
 
 	runner := NewDumpRunner(basePath)
 
-	log.Print("Running tasks...")
-	RunAllTasks(runner, basePath, *concurrentTasks)
+	log.Print("Running dump tasks...")
+	RunAllDumpTasks(runner, basePath, *concurrentTasks)
```

Comment:
Yeah, this confused me too, it's the way the flags module parses the flags to the program.